scormxblock/scormxblock.py

```diff
@@ -8,6 +8,7 @@
 import shutil
 import xml.etree.ElementTree as ET
 
+from completion import waffle as completion_waffle
```

Comment:
no need to add waffle switches in xblock this should be handled by edx completion tool.